.github/workflows/Automation Master SDK Workflow.yml

```diff
@@ -202,8 +215,9 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+
+      - name: readDir
+        run: ls   
```

Comment:
readDir can be removed as it doesn't serve any purpose. 
Fetch-depth need to be readded again as that is needed for SonarCloud.

Reply:
ok